docker/services/uwsgi.d/60_processes.ini

```diff
@@ -0,0 +1,4 @@
+[uwsgi]
+# Default number of processes and threads for the app
+processes = 3
+threads = 2
```

Comment:
This looks like a low number, if we are API/DB intensive 10~15 can run. Although the real number comes out with iterative testing

Reply:
We've also not set enable-threads=true, It used to be false by default

Reply:
Agreed, these will need to be adjusted as we run the service. I'd like to be able to use the [uwsgi cheaper subsystem](https://uwsgi-docs.readthedocs.io/en/latest/Cheaper.html) to dynamically scale the number of worker processes that we are running, but the large amount of work we do when bringing a new worker online when we create a new [CirculationManager](https://github.com/ThePalaceProject/circulation/blob/881d3b5815b18721212b7d4eb7f14956efac597b/api/controller.py#L130-L147) object means it takes a long time to start a new worker, which limits our options.

Reply:
> We've also not set enable-threads=true, It used to be false by default

When uwsgi is starts with this configuration i see this in the logs: `python threads support enabled`. I believe that means we don't need to set `enable-threads`.